packages/react/src/patterns/blocks/ContentBlockSegmented/__stories__/ContentBlockSegmented.stories.js

```diff
@@ -0,0 +1,175 @@
+import './index.scss';
+import { select, text, withKnobs } from '@storybook/addon-knobs';
+import ContentBlockSegmented from '../ContentBlockSegmented';
+// import ContentGroupSimpleKnobs from '../../ContentGroupSimple/__stories__/data/ContentGroupSimple.knobs';
+import React from 'react';
+import readme from '../README.md';
+import { settings } from 'carbon-components';
+import { storiesOf } from '@storybook/react';
+
+const { prefix } = settings;
+
+storiesOf('Patterns (Blocks)|ContentBlockSegmented', module)
+  .addDecorator(withKnobs)
+  .addParameters({
+    readme: {
+      sidebar: readme,
+    },
+  })
+  .add('Default', () => {
+    const heading = text('Heading', 'Lorem ipsum dolor sit amet.');
+
+    const image = {
+      images: [
+        {
+          src: 'https://fpoimg.com/320x180?bg_color=0f62fe&text_color=ffffff',
+          minWidth: 320,
+        },
+        {
+          src: 'https://fpoimg.com/400x225?bg_color=0f62fe&text_color=ffffff',
+          minWidth: 400,
+        },
+        {
+          src: 'https://fpoimg.com/672x378?bg_color=0f62fe&text_color=ffffff',
+          minWidth: 672,
+        },
+      ],
+      alt: 'CTA image',
+      defaultImage:
+        'https://fpoimg.com/672x378?bg_color=0f62fe&text_color=ffffff',
```

Comment:
BTW, these images are looking a lot better with the updated styles. Thanks for doing that.